### ISSUE TITLE
Support for Flex sensors?

### ISSUE THREAD

**Issue by jcgillespie** [Cisien/homebridge-sense-power-meter#4]
Does this plugin support the [Sense Flex Sensors](https://help.sense.com/hc/en-us/articles/360053561974-What-are-Sense-Flex-Sensors-)? I have the CT sensors for solar monitoring and would like to report consumption AND production.

**Comment by Cisien** [comment]
I don't have access to one of these sensors, so I can't answer for sure. This plugin connects to the same sense APIs that the web browser uses, so in theory it's at least partially compatible. Please give it a try and let me know if you run into issues.

**Comment by audientsoundco** [comment]
I was able to display the solar output in the log (modifying my local copy of senseAccessory.js). It seems the api supports it. the next question is what can one do with that info? 

can one plugin return 2 sensor values, or does one create a second accessory?

**Comment by jcgillespie** [comment]
I imagine it would create a second, configurable accessory.  The value it exposes is dependent on how that sensor is utilized. They can be set up to monitor solar production, a single dedicated circuit, generator production, or 400A split-service panels.

**Comment by audientsoundco** [comment]
right, it's another quantity that Sense can report on, my added sensor happens to be attached to a solar array. presumably homebridge can be used to track whatever you plug the aux sensor into. the info is making it to homebridge, dynamically updated, but as yet, it's not displayed in any widget, or made visible to HomeKit. 

How cool would it be to have a HomeKit automation trigger on "when my solar array output exceeds X kilowatts..."
